tests/test-cmd/test-build/test.build.js

```diff
@@ -0,0 +1,103 @@
+import path from 'path';
+import {assert} from 'chai';
+
+import {safeFileName} from '../../../src/cmd/build';
+import {onlyInstancesOf, InvalidManifest} from '../../../src/errors';
+import {withTempDir, ZipFile} from '../../helpers';
+import * as fs from '../../../src/util/promised-fs';
+import {getPackageBasename, prepareBuildDir} from '../../../src/cmd/build';
+
+import * as adapter from './adapter';
+
+
+describe('build', () => {
+
+  describe('build', () => {
+
+    it('zips a package', () => {
+      let zipFile = new ZipFile();
+
+      return withTempDir(
+        (tmpDir) =>
+          adapter.buildMinimalExt(tmpDir)
```

Comment:
omgyes! This is way more concise. I think we should convert the entire code base to this so I filed an issue for it: https://github.com/mozilla/web-ext/issues/72